.github/workflows/publish_latest_server_image.yml

```
@@ -0,0 +1,97 @@
+name: "Publish latest server docker image automatically"
+# todo: expand the yaml to deal with multi release version 
+on:
+  schedule:
+    - cron: '0 23 * * *'
+  workflow_dispatch:
+
+jobs:
+  build_latest:
+    runs-on: ubuntu-latest
+    env:
+      repository_url: apache/incubator-hugegraph
```

to avoid name changed in future
```suggestion
      repository_url: apache/hugegraph
```